GovUkDesignSystem/GovUkDesignSystemComponents/DateInputItem.cshtml

```diff
@@ -11,7 +11,6 @@
                name="@(Model.Name)"
                type="number"
                value="@(Model.Value)"
-               pattern="@(Model.Pattern)"
```

Comment:
Why are we removing this?

Reply:
In release note for 4.1.0 of the Frontend it recommends "Remove pattern from the date input component". However, looking more carefully the related patch only removes the default value for the pattern, so I'll update this to match.